src/FwdState.cc

```diff
@@ -837,7 +840,7 @@ FwdState::noteConnection(HappyConnOpener::Answer &answer)
             });
     }
 
-    secureConnectionToPeerIfNeeded(answer.conn);
+    secureConnectionToPeerIfNeeded(static_cast<Comm::ConnectionPointer>(answer.conn));
```

Comment:
Can we remove this explicit cast? Why does not the existing casting operator of `answer.conn` (i.e. PeerConnectionPointer) work here?

Reply:
Removed.